presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PredicatePushDown.java

```diff
@@ -877,30 +874,6 @@ public PlanNode visitSemiJoin(SemiJoinNode node, RewriteContext<Expression> cont
             return output;
         }
 
-        private PlanNode rewriteSemiJoinFilteringSource(SemiJoinNode node, RewriteContext<Expression> context, Expression inheritedPredicate, Expression sourceEffectivePredicate)
```

Comment:
This method was extracted a few commits above, which is kind of pointless since it's being removed here. Squash it with this commit and do the removal in one step.